uvicorn/protocols/http/h11_impl.py

```diff
@@ -266,6 +266,11 @@ def handle_upgrade(self, event):
         if upgrade_value != b"websocket" or self.ws_protocol_class is None:
             msg = "Unsupported upgrade request."
             self.logger.warning(msg)
+            from uvicorn.protocols.websockets.auto import AutoWebSocketsProtocol
```

Comment:
(Nit) Let's add empty lines around this block so it's clearer that it's related but separate from the rest of the error handling code?

Reply:
seems like we're gtg :)